lib/diagnostic.rb

```diff
@@ -29,40 +31,43 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{character} is really a Sith Lord.'
 
 # Define a predicate method in Ruby
 #
 # Define a method that checks whether a number passed in is odd. Name it
 # conventionally.
 
-# define your method here
+def is_odd(number)
+  if !(number % 2 == 0)
+  puts number
+end
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = number.is_odd
```

Comment:
Correct but your math was wrong. If `number % 2 == 0`, then it would be even. :-/